Tmain/json-output-format.d/stdout-expected.txt

```diff
@@ -16,7 +16,7 @@
 {"_type": "tag", "name": "main", "path": "input.go", "pattern": "/^package main$/", "language": "Go", "line": 1, "kind": "package"}
 # json --languages=+man --fields=* --extras=*-{subword}
 {"_type": "ptag", "name": "JSON_OUTPUT_VERSION", "path": "0.0", "pattern": "in development"}
-{"_type": "ptag", "name": "TAG_FILE_FORMAT", "path": "2", "pattern": "extended format; --format=1 will not append ;\" to lines"}
```

Comment:
I wonder TAG_FILE_FORMAT in json output is needed or not. Maybe it is not needed.

Reply:
It's probably interesting to have a versioned format for the JSON output too (e.g. if one day wanting to change the layout), but it probably doesn't make sense to use the same one as for the regular format, as it's totally different and changes to one wouldn't affect the other.

Reply:
Yes.
We have JSON_OUTPUT_VERSION!
So TAG_FILE_FORMAT doesn't make sense. I will remove it in another pull request.